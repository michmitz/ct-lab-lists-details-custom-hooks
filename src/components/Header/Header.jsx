import React from 'react';

export default function Header() {
  return <h1>Rick and Morty Characters Galore!</h1>;
}