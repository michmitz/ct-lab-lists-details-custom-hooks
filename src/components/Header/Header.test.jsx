import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Header from './Header';
import { MemoryRouter } from 'react-router-dom';

describe('Header component', () => {
  afterEach(() => cleanup());
  it('renders Header', () => {
    const { asFragment } = render(
    <MemoryRouter>
    <Header />
    </MemoryRouter>);
    expect(asFragment()).toMatchSnapshot();
  });
});