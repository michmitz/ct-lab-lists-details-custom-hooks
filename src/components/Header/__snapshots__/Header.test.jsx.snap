// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders Header 1`] = `
<DocumentFragment>
  <h1>
    Rick and Morty Characters Galore!
  </h1>
</DocumentFragment>
`;
